jgiven-html-app/src/js/util.js

```diff
@@ -144,3 +144,65 @@ export function deselectAll (options) {
     option.selected = false;
   });
 }
+
+function getArgumentList(wordList) {
+  function isArgument(word) { return !!word.argumentInfo; }
+  return _.chain(wordList)
```

Comment:
could you please check whether the use of the `chain` method has increased the artifact size? There seem to be a problem with that function: https://medium.com/making-internets/why-using-chain-is-a-mistake-9bc1f80d51ba#.4s9828vud